src/components/Chat/MembersSider/Member/index.js

```diff
@@ -4,13 +4,15 @@ import { Link } from 'react-router-dom'
 import './styles.scss'
 
 const Member = props => {
-  const { userId, userName, currentUserId, iconClassNameList } = props
+  const { userId, userName, currentUserId, iconClassNameList, iconMemberStatus } = props
   const userFirstLetter = userName[0]
+  console.log(props)
   return (
     <div className="chat-header-members-sider-member">
       <i aria-hidden="true" title={userFirstLetter} />
       <span className={iconClassNameList}>{userFirstLetter}</span>
-      {currentUserId !== userId && (
+      <span className={iconMemberStatus} ></span>
```

Comment:
span is non-semantic element - if possible, use an element with a semantic meaning.
Empty components are self-closing: `<span className={...} />`